Misc/HISTORY

```diff
@@ -31454,7 +31454,7 @@ aimed at a web server, checks that server for dead links.  Available
 are a command line utility as well as a Tkinter based GUI version.  In
 Tools/webchecker.  A simplified version of this program is dissected
 in my article in O'Reilly's WWW Journal, the issue on Scripting
-Languages (Vol 2, No 2); Scripting the Web with Python (pp 97-120).
+Languages (Vol 2, No 2); Scripting the web with Python (pp 97-120).
```

Comment:
It is the title of the book. It should not be changed until a new edition with changed title be published (and page numbers should be corrected in that case).

Reply:
I didn't notice that it's a title :facepalm: , sorry.